src/content/blog/astro-330.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: "Astro 3.3: <Picture /> component"
+description: "Astro 3.3 is out with a brand new, experimental `<Picture />` component, improved compatability for syntax highlighting, and package provenance."
+publishDate: "October 12, 2023"
+authors:
+  - nate
+  - erika
+  - bjorn
+  - ema
+coverImage: "/src/content/blog/_images/astro-320/blog-hero-3_2-release.webp"
+socialImage: "/src/content/blog/_images/astro-320/og-image-3_2-release.webp"
+lang: "en"
+---
+
+Astro 3.3 is out with a brand new, experimental `<Picture />` component, improved compatability for syntax highlighting, package provenance, and a handful of other quality of life improvements.
+
+- **[New `<Picture />` component](#picture-component)**
+- **[Syntax Highlighting improvements](#syntax-highlighting)**
+- **[Package Provenance](#package-provenance)**
+- **[Quality of Life Improvements](#quality-of-life-improvements)**
+
+```
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Picture Component
+
+When we announced [Better Images in Astro](/blog/images/) with our built-in image optimization feature, we committed to taking your feedback about the feature seriously.
+
+Well, after [a comprehensive RFC](https://github.com/withastro/roadmap/pull/715) and [the most highly upvoted PR](https://github.com/withastro/astro/pull/8620) in Astro's history, we are incredibly excited to introduce the built-in `<Picture />` component!
```

Comment:
```suggestion
When we announced [Better Images in Astro](/blog/images/) with our built-in image optimization feature, we were proud to share our new vision for working with images. In 3.0 we removed our old image integration, and along with it some existing features we know you were using, to focus on building back better. We promised our image story was just getting started.

After [a comprehensive RFC](https://github.com/withastro/roadmap/pull/715) and [the most highly upvoted PR](https://github.com/withastro/astro/pull/8620) in Astro's history, we are incredibly excited to introduce a built-in `<Picture />` component!
```
OK, this might be a little much, but here's *a* kind of take. What do you think of working off a feeling like this?

Reply:
I like it a lot!